src/components/OrderBookWidget.tsx

```diff
@@ -146,19 +146,20 @@ const processData = (
   const quoteTokenDecimals = quoteToken.decimals
   const baseTokenDecimals = baseToken.decimals
 
-  // Convert RawPricePoint into PricePoint:
-  //  Raw items use number (floats) and are given in "atoms"
-  //  Normalized items use decimals (BigNumber) and are given in natural units
-  let pricePoints: PricePoint[] = rawPricePoints.map(pricePoint =>
-    _toPricePoint(pricePoint, quoteTokenDecimals, baseTokenDecimals),
-  )
-
-  // Filter tiny orders
-  pricePoints = pricePoints.filter(pricePoint => pricePoint.volume.gt(SMALL_VOLUME_THRESHOLD))
-
   // Convert the price points that can be represented in the graph (PricePointDetails)
-  const { points } = pricePoints.reduce(
-    (acc, pricePoint, index) => {
+  const { points } = rawPricePoints.reduce(
+    (acc, rawPricePoint, index) => {
+      // Convert RawPricePoint into PricePoint:
+      //  Raw items use number (floats) and are given in "atoms"
+      //  Normalized items use decimals (BigNumber) and are given in natural units
+      const pricePoint = _toPricePoint(rawPricePoint, quoteTokenDecimals, baseTokenDecimals)
+
+      // Filter tiny orders
+      if (pricePoint.volume.lte(SMALL_VOLUME_THRESHOLD)) {
+        logDebug('[OrderBook] pricePoint discarded: ', pricePoint.price.toNumber(), pricePoint.volume.toNumber())
+        return acc
+      }
```

Comment:
Somehow this is the problem. If you just remove this condition everything works
I don't understand why :confounded: 

If you prefiler like on L157 -- works, if you postfilter `points` before returning from function -- works.
But inside `reduce` -- doesn't..

Reply:
Okay, thanks Dima. I think I'll close this PR.

I don't get it, I see the signature of:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce

The callback returns the reduced object. If I ignore the point, I just return `acc` then.
Also, I printed the results, and it's equivalent in the console (I text compared the result) 🤷‍♂️

I'll close